jmclient/jmclient/support.py

```diff
@@ -226,6 +227,26 @@ def cheapest_order_choose(orders, n):
     """
     return orders[0]
 
+def fidelity_bond_weighted_order_choose(orders, n):
+    """
+    choose orders based on fidelity bond for improved sybil resistance
+
+    * with probability `bondless_makers_allowance` will revert to previous default
+      order choose (random_under_max_order_choose)
+    * with probability `1 - bondless_makers_allowance` if there are no bond offerings, revert
```

Comment:
I think it needs a colon between `... 1-bondless_makers_allowance` and ` if there are ..`